crates/scroll/evm/src/lib.rs

```diff
@@ -126,3 +126,18 @@ pub fn spec_id_at_timestamp_and_number(
         ScrollSpecId::SHANGHAI
     }
 }
+
+/// The attributes for the next block env.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ScrollNextBlockEnvAttributes {
+    /// The timestamp of the next block.
+    pub timestamp: u64,
+    /// The suggested fee recipient for the next block.
+    pub suggested_fee_recipient: Address,
+    /// The randomness value for the next block.
+    pub prev_randao: B256,
```

Comment:
I see `parent_beacon_block_root` and `withdrawals` were dropped, why do we keep `prev_randao`?